Kajabi/sage-lib
# Alert: Sync to latest designs

## Description

This PR updates Alert to match latest design specs.

- [x] Heading icon aligns with heading text
- [x] Button and link colors use default colors and are not customizable
- [x] Ensure second link option is available in the button group

In addition, I documented some quesion and recommendations for team discussion to further improve this component: https://kajabi.atlassian.net/browse/DSS-169

I also have a follow up PR for cleaning up Rails documentation and welcome early feedback: https://github.com/Kajabi/sage-lib/pull/1600

## Screenshots

[Figma specs](https://www.figma.com/file/sMbtLUHSt2vfKgKjyQ3052/%F0%9F%A7%A9-Sage-components?node-id=13478%3A24453)

| | Before | After |
|---|---|---|
| Rails regular | ![Screen Shot 2022-09-30 at 12 05 44 PM](https://user-images.githubusercontent.com/17955295/193312180-9b025e2c-7d3b-4934-9398-452cb9042ad5.png) | ![Screen Shot 2022-09-30 at 12 06 06 PM](https://user-images.githubusercontent.com/17955295/193312239-dc8826c4-5858-43b5-8f5e-b1e7be2e8286.png) |
| Rails small | ![Screen Shot 2022-09-30 at 12 05 53 PM](https://user-images.githubusercontent.com/17955295/193312207-37e3bfbf-f305-4db7-b3fd-0b308892ef52.png) | ![Screen Shot 2022-09-30 at 12 06 13 PM](https://user-images.githubusercontent.com/17955295/193312279-a8fc9f5d-48d5-40c4-8572-0814d8d7bf35.png) |
| React regular | ![Screen Shot 2022-09-30 at 12 05 04 PM](https://user-images.githubusercontent.com/17955295/193312148-80b5e1f0-f749-45db-b9b2-d9d0f561a678.png) | ![Screen Shot 2022-09-30 at 12 07 19 PM](https://user-images.githubusercontent.com/17955295/193312337-b4ed0bcc-8c53-477a-926c-f0efabddf0f8.png) |
| React small | ![Screen Shot 2022-09-30 at 12 04 56 PM](https://user-images.githubusercontent.com/17955295/193312124-60750e19-26a7-4c98-9e13-6aed522d7d7f.png) | ![Screen Shot 2022-09-30 at 12 06 55 PM](https://user-images.githubusercontent.com/17955295/193312375-8a8d5326-761d-4343-b58a-3758fe1780e0.png) |

## Testing in `sage-lib`

- See http://localhost:4000/pages/component/alert?tab=preview
- See http://localhost:4100/?path=/docs/sage-alert--dismissable-alert

## Testing in `kajabi-products`

1. (**LOW**) Update Alert to latest design specs.


## Related

https://kajabi.atlassian.net/browse/DSS-143
